Sentence casing (#1037)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-snapshotting-source-freshness.md

---
title: "Source freshness"
id: "cloud-snapshotting-source-freshness"

> description: "Look at a recent snapshot to see the state of source data."

Reviewer: "Snapshotting" terminology is actually deprecated (we have another feature called snapshots and this was confusing - see [the info box on this page](https://docs.getdbt.com/reference/commands/source)). This might need a wider re-work?
```suggestion
description: "Validate that data freshness meets expectations and alert if stale."
```

> description: "You can see the state of source data by looking at a recent snapshot."

Author: ```suggestion
description: "Look at a recent snapshot to see the state of source data."
```
